tests/handlers/test_oidc.py

```diff
@@ -855,69 +854,60 @@ def _generate_oidc_session_token(
         )
 
 
+@skip_unless(HAS_OIDC, "requires OIDC")
```

Comment:
I saw we made this change in other places too -- is there a benefit to `skip_unless`? Or just a bit simpler?

Reply:
just simpler.